administration/command-line-interface.md

```diff
@@ -34,62 +35,128 @@ docker-compose run --rm cli azuracast_cli [command]
 php /var/azuracast/www/bin/console [command]
 ```
 
-# Command Reference
+# Account Related CLI Commands
+This section will cover all account related CLI commands and the response the CLI will provide.  
 
-<br>
+## List all accounts
+```
+(cli_command) azuracast:account:list 
 
-## List All Commands
+AzuraCast User Accounts
+=======================
 
-To see all commands available, run the appropriate CLI command for your installation type with no command specified.
+ ---------------- ------ --------------------- -------------------
+  E-mail Address   Name   Roles                 Created
+ ---------------- ------ --------------------- -------------------
+  name@domain.com         Super Administrator   YYYY-MM-DD HH:MM
+ ---------------- ------ --------------------- -------------------
+```
 
-<br>
+## Create a unique login recovery URL for the specified account.
+``` 
+(cli_command) azuracast:account:login-token (EMAIL)
```

Comment:
In most places where we have placeholders in examples we have them in the format of `[example]` so I think we should keep it that way. There are a few instances here in these changes were the format would need to be corrected.

Also you might want to put this one here into lower-case to match the text description further down (`[email]` instead of `[EMAIL]`)

Reply:
I'll be getting the email reference thing changed in a moment

For clarification, do you request for strings such as: `(cli_command) azuracast:settings:set (setting-key) (setting-value)` to be changed to `(cli_command) azuracast:settings:set {example} {example}`? I may be misunderstanding you here.

Reply:
> For clarification, do you request for strings such as: (cli_command) azuracast:settings:set (setting-key) (setting-value) to be changed to (cli_command) azuracast:settings:set {example} {example}? I may be misunderstanding you here.

They should be `(cli_command) azuracast:settings:set [example] [example]` with square brackets `[]` not `{}`

Reply:
Thanks, done in the last commit.